src/main/java/com/coronacarecard/config/ApiExceptionControllerAdvice.java

```diff
@@ -37,22 +51,22 @@
     }
 
     @ExceptionHandler(InternalException.class)
-    protected ResponseEntity<Object> handleInternalExceptions(InternalException exp) {
+    protected ResponseEntity<Object> handleInternalException(InternalException exp) {
         ApiError error = new ApiError();
         error.code = 500;
         error.status = HttpStatus.INTERNAL_SERVER_ERROR.toString();
-        error.message = "Uh oh! something went wrong processing your request. Please try again or advise the administrator.";
+        error.message = exp.getMessage();
```

Comment:
I dont think we should show the customer intneral message. The previous approach was better. Also i added some code to log this exception since we are overriding it